### ISSUE TITLE
Allow keyboard arrows to move up/down and Del key to remove read feeds – in Two panels (Fox memory) theme

### ISSUE THREAD

**Issue by defenset** [dauphine-dev/drop-feeds#134]
**Describe the solution you'd like**  
I’d would make navigation/managing of feeds a lot easier, moving up and down with arrows and deleting unwanted/read feeds 

**Additional context**  
This feature request is for the Two panels (Fox memory) theme

**Comment by dauphine-dev** [comment]
Hi @defenset,
Thank you for your proposal.
I will see if I can do that, in the next version.

**Comment by defenset** [comment]
great, thanks for looking into it

**Comment by dauphine-dev** [comment]
I have made a 1st version, with navigation by arrow keys.
Available only on github for now.

You can get it from this link :
https://github.com/dauphine-dev/drop-feeds/releases/tag/1.0.14beta1
Tell me what do think about it.

I not sure what you want to say with "deleting unwanted/read feeds". You want an option to hide feeds already read ?

**Comment by defenset** [comment]
1. Navigation through keyboard arrows works great! I just tested it. Excellent!

2. > I not sure what you want to say with "deleting unwanted/read feeds". You want an option to hide feeds already read ?

        Yes, hide unwanted aka not being visible, 
        feeds that are not yet read are highlighted, 
        ones that has been read are not highlighted anymore
        and the ones that someone want to delete would not be visible anymore, 
        would be hidden from the list of feeds
        (if that’s possible of course)

**Comment by dauphine-dev** [comment]
Hi @defenset 
There is a new beta version with an option to hide read articles : 
[https://github.com/dauphine-dev/drop-feeds/releases/tag/1.0.14beta2](https://github.com/dauphine-dev/drop-feeds/releases/tag/1.0.14beta2)

Please tell me if you found any issues.

**Comment by defenset** [comment]
When you scroll through feeds with arrow keys and press one more time “go up key” or one more time “go down key” when on last feed, “highlighting” disappears and won’t come back again, unless you bring it back via mouse click.

And del key is not deleting feeds so far for me.

**Comment by dauphine-dev** [comment]
We had an understanding.
I will change that next time.

For now read articles are hidden when the button in the top left corner is set. To unhide read articles unset the button.

**Comment by defenset** [comment]
Would it be possible to make DEL key remove entries from main panel ?

Like it does in the left side box ? See screenshot

**Comment by defenset** [comment]
![feeds del](https://user-images.githubusercontent.com/94124619/143039615-83abc43d-acbc-4088-b2a7-93a91578c276.png)

**Comment by dauphine-dev** [comment]
Yeah agree totally. I was thinking to bring same icons/features from the article panel to the fox_memory render view.

**Comment by defenset** [comment]
Screenshot for earlier issue, Nav keys behavior
![bar](https://user-images.githubusercontent.com/94124619/143255695-8d8eed1f-a41f-41d4-b497-7a843782afae.png)
